### PR TITLE
Adds navbar stuff

### LINKED ISSUE #33: decide what should be on the navbar
### description
what categories or pages do we want linked in our top navigation

### implementation?
check code in
#13

### how to validate?
pull down branch, run and manually test by at least 1 reviewer before merge

### dependencies?
- [ ] #12 (create pages that navbar links to)

### PR DESCRIPTION
**Pull request by ghambhackmud**
This is a pr for the navbar. It adds 1 depth navigation at the top.

This will not work correctly until place holder articles or the example ones exist. [examples](https://github.com/comcode-org/hackmud_wiki/pull/12) would fix this, so it'd have to be merged first.

fixes #33

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -75,10 +75,46 @@ const config = {
...
+              sidebarId: "scriptingSidebar",
+            },
+          ],
+        },
+      ],
     },
     footer: {
-      copyright:
-        'Licensed by ComCODE with <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank" rel="noopener noreferrer">CC BY-NC-SA 4.0</a> for creative and educational use.',
+      copyright: `Copyright © ${new Date().getFullYear()} ComCODE`,
```
Comment (seanmakesgames) [inline docusaurus.config.js:116]: todo undo this change

**Review by seanmakesgames — CHANGES_REQUESTED** [review]
unintentional change of footer

File: docusaurus.config.js (hunk 1 above)
Comment (seanmakesgames) [reply docusaurus.config.js:116]:
```suggestion
      copyright:
        'Licensed by ComCODE with <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank" rel="noopener noreferrer">CC BY-NC-SA 4.0</a> for creative and educational use.',
```

**Review by seanmakesgames — APPROVED** [review]

**Review by tukib — APPROVED** [review]
